<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuWidg</class>
 <widget class="QWidget" name="MenuWidg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="logIn_btn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Авторизоваться</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loan_btn">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>130</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Взять кредит</string>
   </property>
  </widget>
  <widget class="QPushButton" name="savings_btn">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>130</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Открыть вклад</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
